packages/common/mui/src/lib/product-dashboard/components/dashboard-card.tsx

```diff
@@ -161,10 +171,23 @@ const GridCard = styled(Box)(
         `
 );
 
-const GridCardTitle = styled(SmallInput)(
-  `
+const GridCardTitle = styled(H4)(
+  ({ theme }) => `
+    font-family: Avenir Next;
     display: block;
     text-align: left;
+    margin-bottom: ${theme.spacing(0.5)};
+      `
+);
+
+const SectionTitle = styled(Box)(
```

Comment:
Is there no theme style for this?

Reply:
The problem is the card/landing page typography style bullshit. We were using that but it breaks a lot in the data table when I used that theme. It was easier to create the couple things we needed for the cards instead of fixing the table

Reply:
ok sounds good